hello.cpp

```diff
@@ -1,28 +1,45 @@
+#include <cstring>
 #include <launchdarkly/api.hpp>
 
 const char *username = "jeff@example.com";
 
 // update me to your own keys
-const char *sdk_key = "YOUR_SDK_KEY";
-const char *feature_key = "YOUR_FEATURE_KEY";
+#define MOBILE_KEY "YOUR_MOBILE_KEY"
+#define FEATURE_KEY "YOUR_FEATURE_KEY"
 
 int main() {
+
+    if (strlen(MOBILE_KEY) == 0) {
+        printf("*** Please edit hello.cpp to set MOBILE_KEY to your LaunchDarkly mobile key first\n\n");
+        return 1;
+    }
+
     LDConfigureGlobalLogger(LD_LOG_INFO, LDBasicLogger);
 
-    auto *config = LDConfigNew(sdk_key);
+    auto *config = LDConfigNew(MOBILE_KEY);
+
+    // Set up the user properties. This user should appear on your LaunchDarkly users dashboard
+    // soon after you run the demo.
     auto *user = LDUserNew(username);
```

Comment:
For the behavior in the spec, we'd like to have the user key be "example-user-key" and the user name be "Sandy". #5 had 
```
    struct LDUser *user = LDUserNew("example-user-key");
    LDUserSetName(user, "Sandy");
```
but I'm not certain that's the best way to declare it.